articles/active-directory/external-identities/b2b-quickstart-invite-powershell.md

```diff
@@ -35,33 +35,32 @@ You need a test email account that you can send the invitation to. The account m
 Run the following command to connect to the tenant domain:
 
 ```powershell
-Connect-AzureAD -TenantDomain "<Tenant_Domain_Name>"
+Connect-MgGraph -Scopes user.readwrite.all
 ```
-For example, `Connect-AzureAD -TenantDomain "contoso.onmicrosoft.com"`.
 
 When prompted, enter your credentials.
 
 ## Send an invitation
 
-1. To send an invitation to your test email account, run the following PowerShell command (replace **"Sanda"** and **sanda\@fabrikam.com** with your test email account name and email address): 
+1. To send an invitation to your test email account, run the following PowerShell command (replace **"John Doe"** and **john\@contoso.com** with your test email account name and email address): 
 
    ```powershell
-   New-AzureADMSInvitation -InvitedUserDisplayName "Sanda" -InvitedUserEmailAddress sanda@fabrikam.com -InviteRedirectURL https://myapps.microsoft.com -SendInvitationMessage $true
+   New-MgInvitation -InvitedUserDisplayName "John Doe" -InvitedUserEmailAddress John@contoso.com -InviteRedirectUrl "https://myapplications.microsoft.com" -SendInvitationMessage:$true
    ```
 2. The command sends an invitation to the email address specified. Check the output, which should look similar to the following:
 
-   ![PowerShell output showing pending user acceptance](media/quickstart-invite-powershell/powershell-azureadmsinvitation-result.png)
+   ![image](https://user-images.githubusercontent.com/49490355/152628050-277376df-cc1f-42c6-8a81-a2dbba66b8d8.png)
 
 ## Verify the user exists in the directory
 
-1. To verify that the invited user was added to Azure AD, run the following command:
+1. To verify that the invited user was added to Azure AD, run the following command (replace **john\@contoso.com** with your invited email):
  
    ```powershell
-   Get-AzureADUser -Filter "UserType eq 'Guest'"
+   Get-MgUser -Filter "Mail eq 'John@contoso.com'"
    ```
-3. Check the output to make sure the user you invited is listed, with a user principal name (UPN) in the format *emailaddress*#EXT#\@*domain*. For example, *sanda_fabrikam.com#EXT#\@contoso.onmicrosoft.com*, where contoso.onmicrosoft.com is the organization from which you sent the invitations.
+3. Check the output to make sure the user you invited is listed, with a user principal name (UPN) in the format *emailaddress*#EXT#\@*domain*. For example, *john_contoso.com#EXT#\@fabrikam.onmicrosoft.com*, where fabrikam.onmicrosoft.com is the organization from which you sent the invitations.
 
-   ![PowerShell output showing guest user added](media/quickstart-invite-powershell/powershell-guest-user-added.png)
+   ![image](https://user-images.githubusercontent.com/49490355/152627709-05e22251-cecc-414e-9897-014bdd7d7c48.png)
```

Comment:
Please also include this screenshot with filename "powershell-mginvitation-guest-user-add.png" (all lowercase) in the media/quickstart-invite-powershell folder and update line 63:

```suggestion
   ![PowerShell output showing guest user added](media/quickstart-invite-powershell/powershell-mginvitation-guest-user-add.png)
```